LLVM.Imports.BitReader.pas

```diff
@@ -9,16 +9,25 @@ interface
   LLVM.Imports.Types;
 
 //LLVMParseBitcode is deprecated, therefore we implement only LLVMParseBitcode2, but as LLVMParseBitcode
-function LLVMParseBitcode(MemBuf: TLLVMMemoryBufferRef; out OutModule: TLLVMModuleRef): TLLVMBool; cdecl; external CLLVMLibrary name 'LLVMParseBitcode2';
+function LLVMParseBitcode(MemBuf: TLLVMMemoryBufferRef; out OutModule: TLLVMModuleRef; var OutMessage: PLLVMChar): TLLVMBool; cdecl; external CLLVMLibrary name 'LLVMParseBitcode2';
+//* Builds a module from the bitcode in the specified memory buffer, returning a  reference to the module via the OutModule parameter. Returns 0 on success. */
+function LLVMParseBitcode2(MemBuf: TLLVMMemoryBufferRef; out OutModule: TLLVMModuleRef): TLLVMBool; cdecl; external CLLVMLibrary name 'LLVMParseBitcode2';
```

Comment:
This doesn't seem right.
The function is imported with 2 signatures.

Reply:
yes exactly but it remains obsolete so I had eliminated it but then I reinserted it for compatibility.